main.go

```diff
@@ -77,7 +77,10 @@ func main() {
 	}
 
 	if len(todoErrs) > 0 {
-		printTodoErrs(todoErrs, *format)
+		err = printTodoErrs(todoErrs, *format)
+		if err != nil {
+			logger.Info("Error when printing todo errors" + err.Error())
```

Comment:
This should be a panic as we should never get here

Reply:
I changed the code to panic if this point is reached during execution.